config/routes.rb

```diff
@@ -74,36 +86,12 @@
         end
       end
 
-      resources :criteria do
+      resources :criteria, except: [:index] do
         collection do
           post 'update_positions'
         end
       end
 
-      resources :rubric_criteria do
```

Comment:
If you want to eliminate the RESTful routes, you don't need to delete this block entirely (and shouldn't because of the other routes). Instead, do `only: []` to get rid of all the automatically generated routes. This way you won't need to change the `collection` part.